com.unity.multiplayer.mlapi/Runtime/NetworkVariable/NetworkVariable.cs

```diff
@@ -18,14 +18,6 @@ public class NetworkVariable<T> : INetworkVariable
         /// </summary>
         public readonly NetworkVariableSettings Settings = new NetworkVariableSettings();
 
-        /// <summary>
-        /// The last time the variable was written to locally
-        /// </summary>
-        public ushort LocalTick { get; internal set; }
```

Comment:
I don't think we should remove tick here and definitely not now like this. Tick will still be relevant. The new time system will cause us to only have 1 tick value instead of LocalTick/RemoteTick but this is still relevant. While we do not use these  tick properties internally they will be used by future systems like snapshot/interpolation and could be used by our users. Lets have a conversation on how we will transition tick values into our new system before just removing it.

Reply:
I'd still suggest this removal happen before we implement something new.
When the new stuff lands, it may or may not be there (or named differently, located under a different class etc.)

Reply:
The thing is local/write tick information is important to us. It  could help us some of the problems we currently have with client network variable writes where OnValueChanged events get triggered twice. By using the tick we could prevent that. From my understanding we will need to rework all of this with the snapshot/interpolation system which will happen in the next few weeks. So why not do it then?

Reply:
Ah, ok, this made me realize something, that with snapshots NetworkVariable is a NetworkVariable snapshot, and we store an array of them, so then each could have a queryable timestamp?  I guess that would be a convenience so you could look at a NetworkVariable (snapshot) and query what timestamp it went with vs. knowing that from when you queried its "row" from its snapshot container.